cmd/syft/cli/options/relationships.go

```diff
@@ -0,0 +1,13 @@
+package options
+
+type relationshipsConfig struct {
+	FileOwnership        bool `mapstructure:"file-ownership" json:"file-ownership" yaml:"file-ownership"`
```

Comment:
Same comment as the other place this is used. What does `file-ownership: true` or `file-ownership: false` mean?

Reply:
I've renamed by prepepending `package` to each element and adding descriptions.